src/pages/GraphPage.tsx

```diff
@@ -0,0 +1,20 @@
+import { Graph } from '../components/Graph/Graph';
+import { useAuth0 } from '@auth0/auth0-react';
+import { Button } from '@nextui-org/react';
+
+export const GraphPage = () => {
+	const { logout } = useAuth0();
+
+	return (
+		<>
+			<Button
+				color="secondary"
+				css={{ position: 'absolute', width: '200px', left: '15px', margin: '20px', zIndex: 2 }}
```

Comment:
Think of an alternative styling approach within a separate files (CSS modules, styled-components, Sass etc.)